README.md

```diff
@@ -1,130 +1,85 @@
-grtp.co [![Builds][]][Travis] [![Deps][]][Gemnasium]
-=======
+ Gratipay Widgets - [grtp.co][] [![Builds][]][Travis] [![Deps][]][Gemnasium] [![Tips][]][Gratipay]
+================================
 
-Gratipay widgets + widget API
+Gratipay Widgets v2 + v1 API
+
+v1 readme is over at [README.v1.md](README.v1.md).
 
 [Builds]: http://img.shields.io/Travis-ci/gratipay/grtp.co.png "Build Status"
 [Travis]: https://travis-ci.org/gratipay/grtp.co
 [Deps]: https://Gemnasium.com/gratipay/grtp.co.png "Dependency Status"
 [Gemnasium]: https://gemnasium.com/gratipay/grtp.co
+[Tips]: https://img.shields.io/gratipay/gratipay.svg
+[Gratipay]: https://gratipay.com/Gratipay/
+
 
 ## Getting Started
-Quickstart:
 ```
 $ git clone git@github.com:gratipay/grtp.co.git
 $ cd grtp.co
-$ make run
+$ make test run
 ```
 
-Then visit http://localhost:9537/test/
-
+The widget server should then be listening on http://localhost:5000/.
```

Comment:
Standard test port for Node.js projects.  Just makes sense to use the standard, since, iirc, /test/ isn't really what anyone would expect it to be.